charts/seqr/README.md

```diff
@@ -94,7 +94,6 @@ If you need to upgrade the redis version made available by this chart, you can d
 | fullnameOverride | string | `""` | Overrides the fully qualified name of the app, for use in templates |
 | image.pullPolicy | string | `"Always"` | The policy for pulling images |
 | image.repository | string | `"gcr.io/seqr-project/seqr"` | The docker image repository to pull images from |
-| imagePullSecrets | list | `[]` | If needed, you can provide secrets required to retrieve images |
```

Comment:
These are some `helm` provided charts that our deployment does not make use of.

Reply:
when it comes to imagepullsecrets and tolerations, i'd actually argue that it's a bug that the chart isn't using them. Pull secrets and tolerations are both things that are common to need to set in various deployments, so the chart for public consumption should probably allow for it.

Reply:
yep that's also reasonable!